benchmark/agents/common.py

```diff
@@ -92,7 +92,7 @@ def proximity_detection(OGM):
 
 def heading_to_degree(heading):
     # +y = 0 rad. Note the 0 means up direction
-    return np.degrees((heading + math.pi) % (2 * math.pi))
+    return np.degrees(heading % (2 * math.pi))
 
 
 def heading_to_vec(heading):
```

Comment:
For this method if the intention is for this to convert a north bound heading to `x: right` and `y: up` the correction will need to be `- pi/2` or the phase will be off by `pi`.

Reply:
Converting to `x:right`, `y:up` should be `heading + pi/2`, since now is `x: up`, `y:left`.
Now we assume Up is 0 for heading and increase to 360 degree counterclockwise, following in a way as old heading calculation way.

Reply:
Temporarily using this way, later i think a refactorization will be done to keep the calculation to align with  xy-coordinate system。

Reply:
OK